pkg/generators/list.go

```diff
@@ -0,0 +1,29 @@
+package generators
+
+import (
+	"fmt"
+	argoprojiov1alpha1 "github.com/argoproj-labs/applicationset/api/v1alpha1"
+	argov1alpha1 "github.com/argoproj/argo-cd/pkg/apis/application/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ Generator = (*ListGenerator)(nil)
+
+type ListGenerator struct {
+	commonClient client.Client
+}
+
+func NewListGenerator(client client.Client) Generator {
```

Comment:
I don't think the list generator should need a kubernetes client, given the items are a hard-wired list.

Reply:
Removed.